web/. dockerignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 build
-.dockerignore
```

Comment:
I've removed it because of [this](https://codefresh.io/docker-tutorial/not-ignore-dockerignore-2/)